Filed to build project - ClassNotFoundException: org.apache.commons.lang3.StringUtils

After upgrade Gradle and IDEA plugin to version 2.2: my project filed to build =)

```java
FAILURE: Build failed with an exception.
              
* What went wrong:
Execution failed for task ':app:generateDebugSqlDelightInterface'.
> org/apache/commons/lang3/StringUtils
              
* Try:        
Run with --info or --debug option to get more log output.
              
* Exception is:
org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':app:generateDebugSqlDelightInterface'.
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:69)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:46)
        at org.gradle.api.internal.tasks.execution.PostExecutionAnalysisTaskExecuter.execute(PostExecutionAnalysisTaskExecuter.java:35)
        at org.gradle.api.internal.tasks.execution.SkipUpToDateTaskExecuter.execute(SkipUpToDateTaskExecuter.java:64)
        at org.gradle.api.internal.tasks.execution.ValidatingTaskExecuter.execute(ValidatingTaskExecuter.java:58)
        at org.gradle.api.internal.tasks.execution.SkipEmptySourceFilesTaskExecuter.execute(SkipEmptySourceFilesTaskExecuter.java:52)
        at org.gradle.api.internal.tasks.execution.SkipTaskWithNoActionsExecuter.execute(SkipTaskWithNoActionsExecuter.java:52)
        at org.gradle.api.internal.tasks.execution.SkipOnlyIfTaskExecuter.execute(SkipOnlyIfTaskExecuter.java:53)
        at org.gradle.api.internal.tasks.execution.ExecuteAtMostOnceTaskExecuter.execute(ExecuteAtMostOnceTaskExecuter.java:43)
        at org.gradle.execution.taskgraph.DefaultTaskGraphExecuter$EventFiringTaskWorker.execute(DefaultTaskGraphExecuter.java:203)
        at org.gradle.execution.taskgraph.DefaultTaskGraphExecuter$EventFiringTaskWorker.execute(DefaultTaskGraphExecuter.java:185)
        at org.gradle.execution.taskgraph.AbstractTaskPlanExecutor$TaskExecutorWorker.processTask(AbstractTaskPlanExecutor.java:66)
        at org.gradle.execution.taskgraph.AbstractTaskPlanExecutor$TaskExecutorWorker.run(AbstractTaskPlanExecutor.java:50)
        at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:54)
        at org.gradle.internal.concurrent.StoppableExecutorImpl$1.run(StoppableExecutorImpl.java:40)
Caused by: java.lang.NoClassDefFoundError: org/apache/commons/lang3/StringUtils
        at com.squareup.sqldelight.gradle.SqlDelightTask.detailText(SqlDelightTask.kt:146)
        at com.squareup.sqldelight.gradle.SqlDelightTask.message(SqlDelightTask.kt:130)
        at com.squareup.sqldelight.gradle.SqlDelightTask.access$message(SqlDelightTask.kt:44)
        at com.squareup.sqldelight.gradle.SqlDelightTask$execute$2$1.invoke(SqlDelightTask.kt:75)
        at com.squareup.sqldelight.gradle.SqlDelightTask$execute$2$1.invoke(SqlDelightTask.kt:44)
        at com.squareup.sqldelight.gradle.SqlDelightTask.parseThen(SqlDelightTask.kt:119)
        at com.squareup.sqldelight.gradle.SqlDelightTask.access$parseThen(SqlDelightTask.kt:44)
        at com.squareup.sqldelight.gradle.SqlDelightTask$execute$2.execute(SqlDelightTask.kt:71)
        at com.squareup.sqldelight.gradle.SqlDelightTask$execute$2.execute(SqlDelightTask.kt:44)
        at org.gradle.api.internal.changedetection.changes.RebuildIncrementalTaskInputs.doOutOfDate(RebuildIncrementalTaskInputs.java:45)
        at org.gradle.api.internal.changedetection.changes.StatefulIncrementalTaskInputs.outOfDate(StatefulIncrementalTaskInputs.java:46)
        at org.gradle.api.internal.changedetection.changes.RebuildIncrementalTaskInputs.outOfDate(RebuildIncrementalTaskInputs.java:28)
        at com.squareup.sqldelight.gradle.SqlDelightTask.execute(SqlDelightTask.kt:70)
        at org.gradle.internal.reflect.JavaMethod.invoke(JavaMethod.java:75)
        at org.gradle.api.internal.project.taskfactory.AnnotationProcessingTaskFactory$IncrementalTaskAction.doExecute(AnnotationProcessingTaskFactory.java:244)
        at org.gradle.api.internal.project.taskfactory.AnnotationProcessingTaskFactory$StandardTaskAction.execute(AnnotationProcessingTaskFactory.java:220)
        at org.gradle.api.internal.project.taskfactory.AnnotationProcessingTaskFactory$IncrementalTaskAction.execute(AnnotationProcessingTaskFactory.java:231)
        at org.gradle.api.internal.project.taskfactory.AnnotationProcessingTaskFactory$StandardTaskAction.execute(AnnotationProcessingTaskFactory.java:209)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeAction(ExecuteActionsTaskExecuter.java:80)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:61)
        ... 14 more
Caused by: java.lang.ClassNotFoundException: org.apache.commons.lang3.StringUtils
        ... 34 more
```

OS: Windows 10 x64
Android Studio: 2.0.0 beta6
Gradle 2.11
Android gradle plugin: 2.0.0-beta6
SQLDelight: 0.2.2
SQLDelight Android Studio plugin: 0.2.2
Android Studio used jre from latest IDEA 16 Preview
Gradle used Oracle JDK 1.8.0_73 x86

--- a/sqldelight-gradle-plugin/src/main/kotlin/com/squareup/sqldelight/gradle/SqlDelightTask.kt
+++ b/sqldelight-gradle-plugin/src/main/kotlin/com/squareup/sqldelight/gradle/SqlDelightTask.kt
@@ -30,7 +30,6 @@ import com.squareup.sqldelight.validation.SqlDelightValidator
 import org.antlr.v4.runtime.ANTLRInputStream
 import org.antlr.v4.runtime.CommonTokenStream
 import org.antlr.v4.runtime.ParserRuleContext
-import org.apache.commons.lang3.StringUtils
 import org.gradle.api.logging.LogLevel
 import org.gradle.api.tasks.OutputDirectory
 import org.gradle.api.tasks.SourceTask
@@ -143,8 +142,8 @@ open class SqlDelightTask : SourceTask() {
         if (element.start.charPositionInLine > 0) {
           result.append(("%${element.start.charPositionInLine}s").format(""))
         }
-        result.append(("%s\n").format(
-          StringUtils.repeat('^', element.stop.stopIndex - element.start.startIndex + 1)))
+        result.append(("%s\n")
+            .format("^".repeat(element.stop.stopIndex - element.start.startIndex + 1)))
       }
     }
 

